### PR TITLE
[Fix] Added 'beta' alert message 

### PR DESCRIPTION
**Pull request by MaksymShokin**
@sofiiakvasnevska

## Issue
https://github.com/Fliplet/fliplet-studio/issues/6712#issuecomment-665568565

## Description
Added outline style to menu.

## Screencast
alert-info

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko

### CODE REVIEW

File: src/OrgUsageDashboard.vue [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <template>
   <div class="org-usage-dashboard">
+    <Message class="component" type="alert-info" message="<p><strong>This feature is currently in beta.</strong> We are actively refining the functionality and collecting feedback. If you have any questions please <a href='#'>contact us</a>.<p>" />
```
Comment (tonytlwu) [inline src/OrgUsageDashboard.vue]: @MaksymShokin How is the **Contact us** link going to open the live chat feature? It was mentioned in the specification.

**Review by tonytlwu — CHANGES_REQUESTED** [review]

File: src/OrgUsageDashboard.vue (hunk 1 above)
Comment (MaksymShokin) [reply src/OrgUsageDashboard.vue]: @tonytlwu 
My bad. Missed that. Fixed

**Comment by MaksymShokin** [comment]
chat

File: src/OrgUsageDashboard.vue (hunk 1 above)
Comment (MaksymShokin) [reply src/OrgUsageDashboard.vue]: @tonytlwu Type also fixed

**Review by tonytlwu — APPROVED** [review]

**Pull request merged**